### PR TITLE
fix: remove pop-up colors from 'cisternas'

### LINKED ISSUE #137: Remove the popup colors from the 'cisternas' visualization
Remove the following colors:

Image

### PR DESCRIPTION
**Pull request by LeilaFarias**
This PR removes the popup colors from 'cisternas' visualization to avoid confusion with the legend information.

### How to test it:

1. Go to **'/map?name=cisterna&year=general'**
2. Hover over the states.
  
### The new popup should look like this:

image

### CODE REVIEW

**Review by LazaroQueiroz — APPROVED** [review]
nice work!!

File: src/components/MapPopup/MapPopup.tsx [hunk 1]
```diff
@@ -44,13 +46,25 @@ const MapPopup: React.FC<MapPopupProps> = ({
...
+          ))
+        ) : (
+          <List>
+            {areas.map((areaInfo, index) => (
+              <ListItem key={index}>
+                <strong>{areaInfo.area}</strong>
+              </ListItem>
+            ))}
+          </List>
+        )}
```
Comment (RodrigoEC) [inline src/components/MapPopup/MapPopup.tsx]: What do you think of moving the outer if statement to the line 52 alone? I think would make the code more readable. The list style could be added by giving the LineInfo component a prop that tell it should appear.

File: src/components/MapPopup/MapPopup.styles.tsx [hunk 2]
```diff
@@ -64,3 +64,15 @@ export const Color = styled.div<{ color: string; $percent: number }>`
   border: solid thin #cdcdcd;
   margin-left: 0;
 `;
+
+export const List = styled.ul`
+  width: 95%;
```
Comment (RodrigoEC) [inline src/components/MapPopup/MapPopup.styles.tsx]: I personally don't  like when we give such specific values to width - like 95%. I know that this was to solve this issue, right?

image

But to solve that we could add a 

```suggestion
   padding-right: 0.5rem;
   box-sizing: border-box;
```
It's a better way to solve this, it ultimately makes kind of the same thing but it's easier to debug in the future.

File: src/components/MapPopup/MapPopup.styles.tsx [hunk 3]
```diff
@@ -64,3 +64,15 @@ export const Color = styled.div<{ color: string; $percent: number }>`
...
+`;
+
+export const ListItem = styled.li`
+  text-align: left;
+  list-style: square;
+  line-height: 1.6;
+  padding: 0.05rem;
+  font-size: 0.9rem;
```
Comment (RodrigoEC) [inline src/components/MapPopup/MapPopup.styles.tsx]: I would remove this 'cause the first to properties don't make a huge difference and the last one

```suggestion
   font-size: 0.8rem;
```

File: src/components/MapTiff/MapTiff.tsx [hunk 4]
```diff
@@ -205,9 +205,17 @@ const MapTiff = ({
...
+            // const rasterColors = rasterMetadata.fields.imageData[
+            //   year
+            // ]?.imageParams.map((param: any) => param.color);
+
+            const rasterColors =
+              name === "cisterna"
+                ? []
+                : rasterMetadata.fields.imageData[year]?.imageParams.map(
+                    (param) => param.color,
+                  );
```
Comment (RodrigoEC) [inline src/components/MapTiff/MapTiff.tsx]: Actually liked this solution 'cause it reminds us that this should not be a permanent fix to this "cisterna" problem.

**Review by RodrigoEC — CHANGES_REQUESTED** [review]
Commented some possible refactoring for the code :3

File: src/components/MapPopup/MapPopup.tsx (hunk 1 above)
Comment (LeilaFarias) [reply src/components/MapPopup/MapPopup.tsx]: i'm not sure i understand it, do you mean like this?: 
```suggestion
{areas.map((areaInfo: AreaInfo, index) => (
          <LineInfo key={index} isColored={colors.length > 0}>
            {colors.length > 0 ? (
              <>
                <Color color={areaInfo.color} $percent={areaInfo.percent || 45} />
                {areaInfo.percent && (
                  <PercentArea>{areaInfo.percent}%</PercentArea>
                )}
                <TotalArea>{areaInfo.area}</TotalArea>
              </>
            ) : (
              <ListItem>
                <strong>{areaInfo.area}</strong>
              </ListItem>
            )}
          </LineInfo>
        ))}
```

File: src/components/MapTiff/MapTiff.tsx (hunk 4 above)
Comment (LeilaFarias) [reply src/components/MapTiff/MapTiff.tsx]: Actually, that was the intention lol

File: src/components/MapPopup/MapPopup.tsx (hunk 1 above)
Comment (RodrigoEC) [reply src/components/MapPopup/MapPopup.tsx]: Not really, I think we could wrap everything with a <List\> and every <LineInfo\> could be a <\i\> tag and if the colors.length if bigger than 0 we could add the <Color> tag and if the areaInfo exists we could add the PercentArea

**Comment by LeilaFarias** [comment]
Hey team, i merged this branch with main and added the requested changes, also fixed this bug of the new layout that's in beta right now

image

**Review by RodrigoEC — APPROVED** [review]
Great Work Leila!

abbot-2

File: src/components/MapTiff/MapTiff.tsx [hunk 5]
```diff
@@ -197,9 +197,16 @@ const MapTiff = ({ isReduced = false, ...props }: { isReduced?: boolean }) => {
...
+
+            // const rasterColors = rasterMetadata.fields.imageData[
+            //   year
+            // ]?.imageParams.map((param: any) => param.color);
+            const rasterColors =
+              id === "cisterna"
+                ? []
+                : rasterMetadata.fields.imageData[year]?.imageParams.map(
+                    (param) => param.color,
+                  );
```
Comment (cilasmarques) [inline src/components/MapTiff/MapTiff.tsx:209]: Refact this in the future

**Review by cilasmarques — APPROVED** [review]

**Pull request merged**